src/users/mod.rs

```diff
@@ -114,6 +114,21 @@ impl UsersManager {
     UserGet::new(id).send_to(&self.0).await
   }
 
+   /// Retrieve user details. A list of fields to include or exclude may also be specified.
+  ///
+  /// # Arguments
+  /// * `email` - The email of the user to retrieve.
+  /// # Scopes
+  /// * `read:users`
+  /// * `read:user_idp_tokens`
+  pub async fn get_by_email<A, U, S: AsRef<str>>(&self, email: S) -> Result<User<A, U>, Auth0Error>
+  where
+    A: DeserializeOwned + Send + Sync,
+    U: DeserializeOwned + Send + Sync,
+  {
+    GetUserByEmail::new(email).send_to(&self.0).await
```

Comment:
>    mismatched types
  expected reference `&str`
found type parameter `S`rustc[E0308](https://doc.rust-lang.org/error-index.html#E0308)
mod.rs(117, 35): this type parameter
mod.rs(122, 5): arguments to this function are incorrect
user_get.rs(35, 10): associated function defined here